packages/tpetra/core/test/CrsGraph/CrsGraph_UnitTests_Swap.cpp

```diff
@@ -200,18 +202,18 @@ generate_crsgraph(Teuchos::RCP<comm_t>                      comm,
             global_ids.push_back(r.first);
         }
     }
-    #if 0
-    for(size_t i = 0; i < global_ids.size(); i++)
+
+    if(verbose)
     {
-        std::cout << "p=" << comm_rank << " | " << "global_ids[" << i << "] = " << global_ids[i] << std::endl;
+        for(size_t i = 0; i < global_ids.size(); i++)
+        {
+            std::cout << "p=" << comm_rank << " | " << "global_ids[" << i << "] = " << global_ids[i] << std::endl;
+        }
+        std::cout << "p=" << comm_rank << " | " << "row_map = map_t(" << gbl_num_rows << ", " 
+                  << "global_ids.data(), " << lcl_num_rows << ", 0, comm)" << std::endl;
     }
-    #endif
 
     // Create the Row Map
-    #if 0
-    std::cout << "p=" << comm_rank << " | " << "row_map = map_t(" << gbl_num_rows << ", " << "global_ids.data(), "
-              << lcl_num_rows << ", 0, comm)" << std::endl;
-    #endif
     RCP<const map_t> row_map(new map_t(gbl_num_rows, global_ids.data(), lcl_num_rows, 0, comm));
```

Comment:
For future reference, it's nicer to follow Tpetra's existing convention when naming typedefs.  I consider `foo_t` to be reserved by system headers.  `foo_type` (snake case) is good for public typedefs.  Tpetra makes exceptions for internal (nonpublic) typedefs that it uses almost everywhere, like `LO`, `GO`, and `MV`.

Reply:
@mhoemmen I've never seen a style guide for Trilinos... does such a document exist that captures these conventions or is something like `foo_t` being reserved by system headers just a general C++ convention?  If it's the latter, do you have any links to something that lays these out? Where possible, I do prefer to avoid breaking the conventions ;)

Reply:
There's no style guide, but if you read Tpetra classes and tests you'll get a general impression.